todo-vue/src/components/Home.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="home">
+    <header><h1>Todo App</h1></header>
+    <main>
+      <ul>
+        <CardComponent
+          v-for="todo in todos"
+          :key="todo.id"
+          :title="todo.title"
+          :id="todo.id"
+          @setTitle="editTodo"
+          @delete="deleteTodo"
+        />
+      </ul>
+      <form @submit="addTodo">
+        <input id="new-todo-input" type="text" placeholder="add new todo..." />
+      </form>
+    </main>
+  </div>
+</template>
+
+<script>
+import Card from "./Card.vue";
+
+export default {
+  name: "HomeComponent",
+  data() {
+    return {
+      todos: [],
+    };
+  },
+  methods: {
+    addTodo: function (e) {
+      e.preventDefault();
+      const newTodo = {
+        id: this.todos.length + 1,
+        title: e.target[0].value,
+      };
+      this.todos.push(newTodo);
+      e.target[0].value = "";
+    },
```

Comment:
```suggestion
  data() {
    return {
      todos: [],
      title: '',
    };
  },
  methods: {
    addTodo() {
      const newTodo = {
        id: this.todos.length + 1,
        title: this.title,
      };
      this.todos.push(newTodo);
    },
```
앞선 리뷰를 토대로 이렇게 개선해볼 수 있겠네요!